where-to-travel/src/main/webapp/script.js

```diff
@@ -288,20 +288,18 @@ function addPlacesFromDirection(lat, lng, place_candidates) {
         
         function callback(response, status) {
           if (status == 'OK') {
-            for (row of response.rows) {
-              const results = row.elements;
-              for (result of results) {
-                const element = result;
-                if(element.status=='OK') {
-                  //Check if the time is within the +- 20% of the user's requested time
-                  if (element.duration.value <= time + time*0.2 && element.duration.value >= time - time*0.2) {
-                    acceptablePlaces.push({
-                      "name": listPlaces[i].name,
-                      "geometry": listPlaces[i].geometry,
-                      "timeInSeconds": element.duration.value,
-                      "timeAsString": element.duration.text
-                    });
-                  }
+            const results = response.row.elements;
+            for (result of results) {
+              const element = result;
```

Comment:
FYI, when merging with development, this will most likely cause conflicts with what I have changed.